test/pd-rendering-test.js

```diff
@@ -3,16 +3,19 @@ var path = require('path')
   , assert = require('assert')
   , parsing = require('../lib/parsing')
   , pdRendering = require('../lib/pd-rendering')
+  , NEWLINE_REGEX = /\r?\n/;
 
 describe('pd-rendering', function() {
 
   describe('#render', function() {
 
     it('should succeed parsing/rendering all test patches identically', function() {
-      // TODO
```

Comment:
Oops lol, I thought this TODO was inserted by me, but it was inserted by you... I can put it back in if you want.